src/Sets/VPolytope.jl

```diff
@@ -187,7 +187,7 @@ Then we solve the following ``m``-dimensional linear program.
 ```
 """
 function ∈(x::AbstractVector{N}, P::VPolytope{N};
-           solver=default_lp_solver(N)) where {N}
+    solver=default_lp_solver(N)) where {N}
     vertices = P.vertices
```

Comment:
The other formatter changes are fine, but I really do not like changing the indentation of function arguments to the same level as the body. It is totally unclear where the arguments end and the body starts.
(This type of change repeats below.)
Could you undo them? (Or, if easier, redo the changes without formatting.)

Reply:
It's easy to redo changes without formatting, but let me do a quick check whether it is possible to turn off just the setting you've described.

Reply:
Seems like it's doable, it has lots of options really > https://domluna.github.io/JuliaFormatter.jl/dev/
 
But let's discuss it properly in the context on the other task, we need to come up with a custom format that is suitable for the package.